images_config/ks-robot-master.cfg

```diff
@@ -0,0 +1,456 @@
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty multiverse
+deb http://de.archive.ubuntu.com/ubuntu/ trusty-updates multiverse
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty-updates multiverse
+
+## N.B. software from this repository may not have been tested as
+## extensively as that contained in the main release, although it includes
+## newer versions of some applications which may provide useful features.
+## Also, please note that software in backports WILL NOT receive any review
+## or updates from the Ubuntu security team.
+deb http://de.archive.ubuntu.com/ubuntu/ trusty-backports main restricted universe multiverse
+deb-src http://de.archive.ubuntu.com/ubuntu/ trusty-backports main restricted universe multiverse
+
+deb http://security.ubuntu.com/ubuntu trusty-security main restricted
+deb-src http://security.ubuntu.com/ubuntu trusty-security main restricted
+deb http://security.ubuntu.com/ubuntu trusty-security universe
+deb-src http://security.ubuntu.com/ubuntu trusty-security universe
+deb http://security.ubuntu.com/ubuntu trusty-security multiverse
+deb-src http://security.ubuntu.com/ubuntu trusty-security multiverse
+
+## Uncomment the following two lines to add software from Canonical's
+## 'partner' repository.
+## This software is not part of Ubuntu, but is offered by Canonical and the
+## respective vendors as a service to Ubuntu users.
+# deb http://archive.canonical.com/ubuntu trusty partner
+# deb-src http://archive.canonical.com/ubuntu trusty partner
+
+## This software is not part of Ubuntu, but is offered by third-party
+## developers who want to ship their latest software.
+deb http://extras.ubuntu.com/ubuntu trusty main
+deb-src http://extras.ubuntu.com/ubuntu trusty main
+
+EOP
+) > /etc/apt/sources.list
+apt-get update
+apt-get upgrade -y
+
+
+sleep 20
+apt-get clean
+apt-get update
+aptitude install openssh-client=1:5.9p1-5ubuntu1.10
+apt-get install openssh-server -y
+sleep20
+apt-get install aptitude -y
+apt-get install ros-indigo-rosbash -y -f -y
+apt-get update
+
+apt-get install python-ros-* -y -f -y
+sleep 20
+
+apt-get clean
+apt-get update
+apt-get upgrade -y
+apt-get -f install 
+sleep20
+apt-get install ros-indigo-care-o-bot-robot -y -f -y
+sleep20
+apt-get install ubuntu-desktop -y 
+sudo rosdep init
+rosdep update
+
+ 
+echo -e "\n${green}INFO: Setup udev rules${NC}\n"
+
+
+
+https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/udev_rules/udev_cob.sh
+wget -O /etc/init.d/ https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/udev_rules/udev_cob.sh
+update-rc.d udev_cob.sh defaults
+
+echo -e "\n${green}INFO: Setup bash environment${NC}\n"
+sleep 5
+
+wget -O /etc/cob.bash.bashrc https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/cob.bash.bashrc.b
+echo -e "\n${green}INFO: Setup network interfaces${NC}\n"
+sleep 5
+wget -O /etc/network/interfaces https://raw.githubusercontent.com/ipa-nhg-dd/setup_cob4/master/cob-pcs/networkInterfacesB1
```

Comment:
Use the upstream branch:

https://raw.githubusercontent.com/ **ipa320** /setup_cob4/master/cob-pcs/cob.bash.bashrc.b